staging/prometheus-operator/templates/prometheus/rules/mesosphere-rules/etcd-rules.yaml

```diff
@@ -1,10 +1,10 @@
+{{- if and .Values.mesosphereResources.create .Values.mesosphereResources.rules.etcd }}
 # Rules for alerting against etcd, without the etcd alerts: etcdHighNumberOfFailedGRPCRequests
 # Which at the moment are false positives at a rate of 100%
-{{- if and .Values.mesosphereResources.create .Values.mesosphereResources.rules.etcd }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ printf "%s-%s" (include "prometheus-operator.fullname" .) "etcd" | trunc 63 | trimSuffix "-" }}
+  name: {{ printf "%s-%s" "etcd" (include "prometheus-operator.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Comment:
Just so I understand, is this related to the issue?

Reply:
Gah, no. This was me testing something. Thanks for catching it.

Reply:
It probably should have been this way, though. Otherwise the unique part of the name could, conceivably, be pushed out past the truncate.